src/main/java/com/google/roomies/database/NoSQLDatabase.java

```diff
@@ -35,12 +46,17 @@
   public void updateListing(String documentID, Map<String, Object> fieldsToUpdate);
 
   /**
-  * Get a document from database in a map of <key, value>.
+  * Checks if listing with input id exists in database.
+  */
+  public boolean listingExists(String listingId) throws InterruptedException,
```

Comment:
Do we need this method in interface?
May be just create private method in FirebaseDatabase

Reply:
Good point, changed to private method